### PR TITLE
Load style.css after main.css

### PR DESCRIPTION
**Pull request by crftwrk**
This is for users who are still afraid to touch a .scss file. So, they can do overrides in style.css without the `!important` rule.

### CODE REVIEW

**Review by justinkruit — APPROVED** [review]
Smart plan, should have done that before.

**Comment by justinkruit** [comment]
I've also added a small commit myself, to clean up the `$modificated_bootscoreCss` `if` function a bit. You can quickly check that out before the merge.

**Review by crftwrk — COMMENTED** [review]
Awesome! I'm happy when you just hit the merge button.

**Pull request merged**
